plugins/nodes/src/js/columns/NodesTablePublicIPColumn.tsx

```diff
@@ -0,0 +1,19 @@
+import { TextCell } from "@dcos/ui-kit";
+import * as React from "react";
+import { Tooltip } from "reactjs-components";
+import Node from "src/js/structs/Node";
+
+export default function(item: Node) {
+  const publicIps = item.getPublicIps();
+  if (publicIps.length === 0) {
+    return null;
+  }
+  if (publicIps.length === 1) {
+    return <TextCell>{publicIps.join()}</TextCell>;
```

Comment:
I think `join`-ing a single item array is a bit confusing, I would prefer `publicIps[0]`.

Reply:
true, will change.